dllNetwork/server/ServerI2CInstruction.C

```diff
@@ -686,14 +686,23 @@ ssize_t ServerI2CInstruction::iic_write(Handle * i_handle, InstructionStatus & o
     } 
 }
 
-uint32_t ServerI2CInstruction::iic_reset(Handle * i_handle, int i_type)
+uint32_t ServerI2CInstruction::iic_reset(Handle * i_handle, ResetType i_type)
 {
 #ifdef SERVER_PRINT_PERF
     struct timeval start_time;
     struct timeval end_time;
 #endif
 
     uint32_t rc = 0;
+    int l_type = 0;
```

Comment:
I think you should replace all uses of i_type with l_type throughout the rest of the function.

Reply:
Thank you. I'll fix that.